### PR TITLE
chore: Add discord url to top menu

### PR DESCRIPTION
**Pull request by miralize**
### Description
Small change to add a text link to discord. Found it wasnt that visible with the CI image next to it.

### CODE REVIEW

**Comment by jef** [comment]
I'm a fan of DRY, so I don't really like this to be completely honest. Perhaps if we changed the text of the badge, I wouldn't mind that so much.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # nvidia-snatcher [![ci](https://github.com/jef/nvidia-snatcher/workflows/ci/badge.svg)](https://github.com/jef/nvidia-snatcher/actions?query=workflow%3Aci) [![discord](https://img.shields.io/discord/756303724095471617.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/Cyc7nrz)
 
-[FAQ](#FAQ) | [Issues](https://github.com/jef/nvidia-snatcher/issues) | [Wiki](https://github.com/jef/nvidia-snatcher/wiki)
+[FAQ](#FAQ) | [Issues](https://github.com/jef/nvidia-snatcher/issues) | [Wiki](https://github.com/jef/nvidia-snatcher/wiki) | [Discord](https://discord.gg/Cyc7nrz)
```
Comment (jef) [inline README.md:3]:
```suggestion
# nvidia-snatcher [![ci](https://github.com/jef/nvidia-snatcher/workflows/ci/badge.svg)](https://github.com/jef/nvidia-snatcher/actions?query=workflow%3Aci) [![discord](https://img.shields.io/discord/756303724095471617.svg?label=join&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/Cyc7nrz)

[FAQ](#FAQ) | [Issues](https://github.com/jef/nvidia-snatcher/issues) | [Wiki](https://github.com/jef/nvidia-snatcher/wiki)	[FAQ](#FAQ) | [Issues](https://github.com/jef/nvidia-snatcher/issues) | [Wiki](https://github.com/jef/nvidia-snatcher/wiki)
```

Makes something like this: [discord](https://discord.gg/Cyc7nrz)

I don't know the best verbiage, but maybe you can figure that out

**Review by jef — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (miralize) [reply README.md:3]: Sure yep that works! TBH I just found it a little hidden / hard to discover when the badge inline with the heading of the readme. 

Would you be comfortable with addding the change to the badge as above & then a section in the FAQ along the lines of `if you need additional help in getting set up, pop into our discord...` etc...?

Comment (jef) [reply README.md:3]: > TBH I just found it a little hidden / hard to discover when the badge inline with the heading of the readme.

I think this good feedback! I really don't know what a user looks for 😆 Just that I thought it looked slick.

I think adding to the FAQ is a good idea! I also feel like no matter how many time we will post help, someone will miss it. But this should be plenty enough.

**Comment by jef** [comment]
Thanks!

**Pull request merged**